src/Infrastructure/LeanCode.AuditLogs/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LeanCode.AuditLogs;
```

Comment:
I forgot about that earlier, but we should probably go with [this approach](https://github.com/leancodepl/corelibrary/pull/596#discussion_r1363841717).